### PR TITLE
 dovecot2: upgrade to 2.3.7.2 ; updated database variants

### PR DESCRIPTION
**Pull request by BjarneDMat**
modified:   mail/dovecot2-antispam/Portfile
	modified:   mail/dovecot2-sieve/Portfile
	modified:   mail/dovecot2/Portfile

1) new versions for dovecot2 and dovecot2-sieve
2) modified database variants in dovecot2
  a) removed EOLed
  b) added missing
3) obsoleted dovecot2-antispam

###### Type(s)

- [ ] bugfix
- [x ] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6
Xcode 10.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
LGTM, but I'd like another pair of eyes on this.

File: mail/dovecot2/Portfile [hunk 1]
```diff
@@ -43,9 +43,18 @@ patch.pre_args      -p1
...
-                    size    3653354
+checksums           patch-doc-example-config-conf.d-10-master.conf.diff \
+                    rmd160  d8968a93871950264e7e43de6da8635d69676137 \
+                    sha256  8dbf8d13224df2f562f6010bad0f6c947d2cbf8988ff74e0134159cfc465e1d2 \
+                    size    3653398 \
+                    patch-src-master-master-settings.c.diff \
+                    rmd160  d8968a93871950264e7e43de6da8635d69676137 \
+                    sha256  8dbf8d13224df2f562f6010bad0f6c947d2cbf8988ff74e0134159cfc465e1d2 \
+                    size    3653398 \
+                    core-2.3.7.2.tar.gz \
```
Comment (mf2k) [inline mail/dovecot2/Portfile]: Use ${version} here instead of hardcoding.

Comment (pmetzger) [reply mail/dovecot2/Portfile]: Yah, good idea.

Comment (BjarneDMat) [reply mail/dovecot2/Portfile]:
```suggestion
                    core-${version}.tar.gz \
```

**Comment by mf2k** [comment]
Now you just need to resolve the conflicts and squash the commits into 1.

**Comment by BjarneDMat** [comment]
> Now you just need to resolve the conflicts and squash the commits into 1.
Iis it possible to do that her on github -or- do I have to do a new pull-request
I've discovered that the only way to replace/edit a commit seems to be to do `git push --force origin dovecot2`

**Comment by pmetzger** [comment]
All you needed to do was force push to your branch.

**Pull request closed without merging**